app/src/main/java/com/mapbox/mapboxsdk/plugins/testapp/Utils.kt

```diff
@@ -21,7 +21,7 @@ object Utils {
         Style.getPredefinedStyle("Streets"),
         Style.getPredefinedStyle("Outdoors"),
         Style.getPredefinedStyle("Light"),
-        Style.getPredefinedStyle("Dark"),
+        Style.getPredefinedStyle("Dark")
```

Comment:
I ran `./gradlew ktlintCheck`, which is what the CI is running and what was failing.

I actually like the `,` at the end, because adding/removing an element in a list does not change an unrelated line in the diff :innocent:.

Reply:
> I actually like the , at the end, because adding/removing an element in a list does not change an unrelated line in the diff 😇.

I do it as well, _but_ only for parameters that are actually varargs and take more than a fixed amount =)

Reply:
Doesn't `arrayOf` take an arbitrary number of arguments? :thinking:

Reply:
Yes, I was more thinking of something like here:

https://github.com/maplibre/maplibre-plugins-android/pull/34/files/b32d5211c0a1aa672de16300e38e8f478a18eb27#diff-c70a047a276baaa6cab30edd2c836b1e82b87165c8118d45b195504675af8e19L99

Reply:
> Yes, I was more thinking of something like here:

If you think about it, it applies there as well. Say you decide to add an argument to `LatLng` later, then if you already have a `,` at the end of the last line, then that line won't change in the diff.

In Kotlin that's also very interesting because of named parameters: it is quite typical to have multiple optional parameters, and by keeping the comma at the end of the last line, you can add a new parameter value without changing the unrelated line in the diff.